js/src/redux/providers/registry/reducer.js

```diff
@@ -24,7 +24,7 @@ export default (state = initialState, action) => {
       return state;
     }
 
-    return { ...state, reverse: {
+    return { reverse: {
```

Comment:
Would prefer spacey blocks reverse on next line just aligning with what we have. (Plus each entry on it's own line)